Cargo.lock

```diff
@@ -1272,6 +1383,37 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "idna"
+version = "1.0.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "686f825264d630750a544639377bae737628043f20d38bbc029e8f29ea968a7e"
+dependencies = [
+ "idna_adapter",
+ "smallvec",
+ "utf8_iter",
+]
+
+[[package]]
+name = "idna_adapter"
```

Comment:
lets pin this in solar-compiler so that our binary pins it but it's not enforced on the library users

Reply:
done, didn't really affect the lock though